packages/babel-plugin-transform-es2015-template-literals/test/fixtures/normal/tag-loose/actual.js

```diff
@@ -1,3 +0,0 @@
-var foo = bar`wow\na${ 42 }b ${_.foobar()}`;
```

Comment:
did we mean to delete this tests or are the covered already?

Reply:
that should be covered here:

https://github.com/kedromelon/babel/blob/0781ee36777cf885b366d7c3b1f7c0b0315caca1/packages/babel-plugin-transform-es2015-template-literals/test/fixtures/loose/tag/actual.js#L1

Reply:
er, maybe that isn't the same? i might have misunderstood. thought there was some test duplication, but it can be hard to wrap your head around that while working with so many rennamed files 🙃

Reply:
i'll have a chance to inspect this a bit more later. will check and submit a PR for these tests if they were mistakenly removed

Reply:
submitted https://github.com/babel/babel/pull/6169